runtime/dgsh_tee.sh

```diff
@@ -6,12 +6,15 @@ args=${@:3}
 # Set a default DISH_TOP in this directory if it doesn't exist
 PASH_TOP=${PASH_TOP:-$(git rev-parse --show-toplevel)}
 
-cleanup()
-{
-    kill -SIGTERM $dgsh_tee_pid > /dev/null 2>&1
-}
-trap cleanup EXIT
+# TODO: Doable check if this is still needed. Turned off for distributed exection. 
```

Comment:
Could you also add an issue (describing what exactly is happening and why this was disabled)?

Reply:
Added #539